src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java

```diff
@@ -104,26 +159,32 @@ public String getScript() {
     private static String quote(FilePath f) {
         return f.getRemote().replace("$", "`$");
     }
+    
+    // In order for PowerShell to properly read a script that contains unicode characters the script should have a BOM, but there is no built in support for
+    // writing UTF-8 with BOM in Java.  This code writes a UTF-8 BOM before writing the file contents.
+    private static void writeWithBom(FilePath f, String contents) throws IOException, InterruptedException {
+        OutputStream out = f.write();
+        out.write(new byte[] { (byte)0xEF, (byte)0xBB, (byte)0xBF });
+        out.write(contents.getBytes(Charset.forName("UTF-8")));
+        out.flush();
+        out.close();
+    }
 
     private static final class PowershellController extends FileMonitoringController {
         private PowershellController(FilePath ws) throws IOException, InterruptedException {
             super(ws);
         }
-
-        public FilePath getPowershellMainFile(FilePath ws) throws IOException, InterruptedException {
-            return controlDir(ws).child("powershellMain.ps1");
-        }
         
-        public FilePath getPowershellScriptFile(FilePath ws) throws IOException, InterruptedException {
+        public FilePath getPowerShellScriptFile(FilePath ws) throws IOException, InterruptedException {
             return controlDir(ws).child("powershellScript.ps1");
         }
         
-        public FilePath getPowershellWrapperFile(FilePath ws) throws IOException, InterruptedException {
+        public FilePath getPowerShellWrapperFile(FilePath ws) throws IOException, InterruptedException {
```

Comment:
Please avoid gratuitous changes when filing a PR. It just makes the diff impossible to review effectively.

Reply:
Okay, I'll remember that in the future.  I think I just wanted to fix the casing to be consistent since in some places I use "Powershell" and in others "PowerShell".